netlify.toml

```
@@ -7,6 +7,13 @@ NODE_VERSION = "16"
 GATSBY_CONCURRENT_DOWNLOAD = "15"
 GATSBY_CPU_COUNT = "1"
 
+[[redirects]]
+  from = "/"
+  to = "/temurin/releases/"
+  query = {variant = ":variant"}
```

I know this is *very* picky, but the `query=` filters the `from` match, so how about they sit next to each other for readability? Makes no difference to the logic.

```suggestion
  from = "/"
  query = {variant = ":variant"}
  to = "/temurin/releases/"
```